src/nucname.cpp

```diff
@@ -6,8 +6,20 @@
 #ifndef PYNE_IS_AMALGAMATED
 #include "nucname.h"
 #include "state_map.cpp"
+#include <cctype>
+#include <algorithm>
 #endif
 
+/***String used to check for metastable states***/
+std::string pyne::nucname::metastable_states = "mnopqrstuvxyz";
+//Capitalize all letters in the metastable states string
+std::string get_upper_metastable() {
+    std::string upper_metastable = pyne::nucname::metastable_states;
+    std::transform(upper_metastable.begin(), upper_metastable.end(), upper_metastable.begin(),
+                   [](char c){ return std::toupper(c); });
+    return upper_metastable;
+}
+std::string upper_metastable = get_upper_metastable();
```

Comment:
I don't think we need these if we transform the individual character to upper/lower as appropriate.